### DISCUSSION TITLE
[XLSX-Renderer] Render data to specific worksheet

### DISCUSSION THREAD

**Discussion by cvmabilin** [Siemienik/XToolset#181] [Q&A] [answered]
How can i render my data to specific worksheet? i don't see in the documentation if there a way to select worksheet

**Comment by Siemienik** [comment]
You can omit to render into all unwanted sheets by using `#! FINISH` at the end of the first row.

Here I prepared an example: 
https://codesandbox.io/s/xlsx-renderer-check-forked-f4vze?file=/src/App.js

VM: 
```js
const viewModel = {
  someValue: "If you see this string, that means the cell was rendered."
};
```

Template: 
[example-template.xlsx](https://github.com/Siemienik/XToolset/files/6855706/example-template.xlsx)

**Comment by Siemienik** [comment, marked as answer]
Answare above in a comment: https://github.com/Siemienik/XToolset/discussions/181#discussioncomment-1032332

@cvmabilin I converted this issue to discussion, Q&A category.
You can find my answer: https://github.com/Siemienik/XToolset/discussions/181#discussioncomment-1032332

**Comment by cvmabilin** [comment]
Thank you for the quick response :)
